planner/core/explain.go

```diff
@@ -749,41 +744,12 @@ const MetricTableTimeFormat = "2006-01-02 15:04:05.999"
 
 // ExplainInfo implements Plan interface.
 func (p *PhysicalMemTable) ExplainInfo() string {
-	if p.DBName.L != util.MetricSchemaName.L || !infoschema.IsMetricTable(p.Table.Name.L) {
-		return ""
-	}
-
-	e := p.Extractor.(*MetricTableExtractor)
-	if e.SkipRequest {
-		return "skip_request: true"
-	}
-	promQL := GetMetricTablePromQL(p.ctx, p.Table.Name.L, e.LabelConditions, e.Quantiles)
-	startTime, endTime := e.StartTime, e.EndTime
-	step := time.Second * time.Duration(p.ctx.GetSessionVars().MetricSchemaStep)
-	return fmt.Sprintf("PromQL:%v, start_time:%v, end_time:%v, step:%v",
-		promQL,
-		startTime.In(p.ctx.GetSessionVars().StmtCtx.TimeZone).Format(MetricTableTimeFormat),
-		endTime.In(p.ctx.GetSessionVars().StmtCtx.TimeZone).Format(MetricTableTimeFormat),
-		step,
-	)
-}
-
-// GetMetricTablePromQL uses to get the promQL of metric table.
-func GetMetricTablePromQL(sctx sessionctx.Context, lowerTableName string, labels map[string]set.StringSet, quantiles []float64) string {
-	def, err := infoschema.GetMetricTableDef(lowerTableName)
-	if err != nil {
-		return ""
-	}
-	if len(quantiles) == 0 {
-		quantiles = []float64{def.Quantile}
-	}
-	var buf bytes.Buffer
-	for i, quantile := range quantiles {
-		promQL := def.GenPromQL(sctx, labels, quantile)
-		if i > 0 {
-			buf.WriteByte(',')
+	explain := "table:" + p.Table.Name.O
```

Comment:
Please use `strings.Builder`. It's more effcient to concat the string.

Reply:
great, done.